model/admin/db.go

```diff
@@ -0,0 +1,89 @@
+package admin
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/db/bsonutil"
+	"github.com/pkg/errors"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	Collection       = "admin"
+	SystemSettingsID = "0"
+)
+
+var (
+	IdKey            = bsonutil.MustHaveTag(AdminSettings{}, "Id")
+	BannerKey        = bsonutil.MustHaveTag(AdminSettings{}, "Banner")
+	TaskDispatchKey  = bsonutil.MustHaveTag(AdminSettings{}, "TaskDispatchDisabled")
+	RunnerFlagsKey   = bsonutil.MustHaveTag(AdminSettings{}, "RunnerFlags")
+	HostinitKey      = bsonutil.MustHaveTag(RunnerFlags{}, "HostinitDisabled")
+	MonitorKey       = bsonutil.MustHaveTag(RunnerFlags{}, "MonitorDisabled")
+	NotificationsKey = bsonutil.MustHaveTag(RunnerFlags{}, "NotificationsDisabled")
+	AlertsKey        = bsonutil.MustHaveTag(RunnerFlags{}, "AlertsDisabled")
+	TaskrunnerKey    = bsonutil.MustHaveTag(RunnerFlags{}, "TaskrunnerDisabled")
+	RepotrackerKey   = bsonutil.MustHaveTag(RunnerFlags{}, "RepotrackerDisabled")
+	SchedulerKey     = bsonutil.MustHaveTag(RunnerFlags{}, "SchedulerDisabled")
+)
+
+var SettingsQuery = db.Query(bson.M{IdKey: SystemSettingsID})
```

Comment:
this could totally be private.

I wonder how much else we can make private too?